content/documentation/master/batching.md

```diff
@@ -102,7 +102,7 @@ Here is how you might put this in place:
             @Override
             public CompletionStage<List<Object>> load(List<String> keys) {
                 //
-                // we use supplyAsync() of values here for maximum parellisation
+                // we use supplyAsync() of values here for maximum parallelisation
```

Comment:
Usually, In computer science jargons, I have seen usage of _parallelization_ (US and Canada style english) instead of _parallelisation_ (UK and Aus style english)

Reply:
If you look closely you'll notice I'm correcting the 4 character from e to a. Nothing to do with s or z

Reply:
I was only pointing to the usage of 's' instead of 'z' in the word. Everything else is fine.